kubeclient.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop', '= 0.30.0'
   spec.add_dependency 'rest-client'
   spec.add_dependency 'recursive-open-struct', '= 1.0.0'
-  spec.add_dependency 'http', '= 0.9.8'
+  spec.add_dependency 'http', '< 3'
```

Comment:
Can you also keep >= 0.9.8 minimum?  We don't necessarily work with http 0.0.1 :-)